exercises/hamming/hamming_test.rb

```diff
@@ -1,69 +1,29 @@
 require 'minitest/autorun'
 require_relative 'hamming'
 
-# Common test data version: 2.1.0 b5d154b
+# Common test data version: 2.2.0 4c453c8
 class HammingTest < Minitest::Test
   def test_empty_strands
     # skip
     assert_equal 0, Hamming.compute('', '')
   end
 
-  def test_identical_strands
+  def test_single_letter_identical_strands
     skip
     assert_equal 0, Hamming.compute('A', 'A')
   end
 
-  def test_long_identical_strands
-    skip
-    assert_equal 0, Hamming.compute('GGACTGA', 'GGACTGA')
-  end
```

Comment:
I agree we dont need to test both the one letter and multiple letter strands, but isn't testing the long identical strands more expressive than testing the one letter strands?